### ISSUE TITLE
missing OPENMP ifdef in src/msn.c

### ISSUE THREAD

**Issue by zkamvar** [JonahBrooks/poppr#6]
An `#ifdef` statement was missing in [`src/msn.c`](https://github.com/JonahBrooks/poppr/blob/devel/src/msn.c#L45).

The line

```
#include <omp.h>
```

Should be

```
#ifdef _OPENMP
#include <omp.h>
#endif 
```

This will allow it to compile on systems that don't yet have openmp support (I'm looking at you, default OSX clang).

**Comment by JonahBrooks** [comment]
Fixed in commit 4a405605dab15064dd1a067153f4ce0e94acaaae

**Comment by zkamvar** [comment]
Huzzah! It works with clang!

``` r
> devtools::install_github("JonahBrooks/poppr@devel")
Downloading github repo JonahBrooks/poppr@devel
Installing poppr
'/Library/Frameworks/R.framework/Resources/bin/R' --vanilla CMD INSTALL  \
  '/private/var/folders/qd/dpdhfsz12wb3c7wz0xdm6dbm0000gn/T/RtmpgLyX9P/devtools2b172fcd27b6/JonahBrooks-poppr-4a40560'  \
  --library='/Users/zhian/R' --install-tests

* installing *source* package ‘poppr’ ...
** libs
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c bitwise_distance.c -o bitwise_distance.o
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c mlg_clustering.c -o mlg_clustering.o
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c msn.c -o msn.o
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c permut_shuffler.c -o permut_shuffler.o
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c poppr_distance.c -o poppr_distance.o
clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/lib -o poppr.so bitwise_distance.o mlg_clustering.o msn.o permut_shuffler.o poppr_distance.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/zhian/R/poppr/libs
** R
** data
** inst
** tests
** preparing package for lazy loading
Creating a generic function for ‘print’ from package ‘base’ in package ‘poppr’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded
* DONE (poppr)
```
